cmd/git-chglog/main.go

```diff
@@ -126,6 +126,12 @@ func CreateApp(actionFunc cli.ActionFunc) *cli.App {
 			Usage: "Regular expression of tag filter. Is specified, only matched tags will be picked",
 		},
 
+		// sort-by-version
+		cli.BoolFlag{
```

Comment:
This will be needed after the rebase with current `master` and the bump to `urfave/cli/v2`

```suggestion
		&cli.BoolFlag{
```

Reply:
Updated. Should be good to go now 😎

Reply:
Thanks for the review as well!